salt/returners/zabbix_return.py

```diff
@@ -54,9 +55,22 @@ def zbx():
         return False
 
 
-def zabbix_send(key, host, output):
-    cmd = zbx()['sender'] + " -c " + zbx()['config'] + " -s " + host + " -k " + key + " -o \"" + output +"\""
-    __salt__['cmd.shell'](cmd)
+def zabbix_send(key, host, output): 
+    f = open(zbx()['zabbix_config'],'r') 
```

Comment:
Hi @JoaquinVeira - I think I misunderstood your question. In addition to importing `salt.utils.files`, you also need to switch this function over to use `salt.utils.files.fopen`, using the `with` context manager instead of Python's regular `open` and `close` functions.

Can you rework this to use that function?

Reply:
like this?

    with salt.utils.fopen (zbx()['zabbix_config'],'r')

Reply:
or:
    with fopen (zbx()['zabbix_config'],'r') as f:
?

Reply:
The former, but I've tweaked it a bit for spacing and added the `as`:
```
with salt.utils.fopen(zbx()['zabbix_config'],'r') as file_handle:
```